jelares/portfolio/target/portfolio-1/javascript/script.js

```diff
@@ -26,4 +26,11 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
-}
+} 
+
+// window.addEventListener('scroll', function(e) {
+
+//     const target = document.querySelectorAll('.parallax');
+//     console.log(target.style);
+
+// });
```

Comment:
Actually it looks like this file was moved to a different folder as a backup? If so, you could consider just deleting this file, since the original version of the code will be available in source control if you ever need to reference it again.